updates/24/updates.sql

```diff
@@ -127,4 +127,6 @@ INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xe485E2f1bab3
 INSERT INTO assets(identifier, name, type) VALUES("eip155:534352/erc20:0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4", "USD Coin", "C");INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:534352/erc20:0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4", "A", 534352, "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4", 6, NULL);INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:534352/erc20:0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4", "USDC", "usd-coin", "USDC", NULL, 1696919085, NULL);
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:534352/erc20:0x5300000000000000000000000000000000000004", "Wrapped Ether", "C");INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:534352/erc20:0x5300000000000000000000000000000000000004", "A", 534352, "0x5300000000000000000000000000000000000004", 18, NULL);INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:534352/erc20:0x5300000000000000000000000000000000000004", "WETH", "weth", "weth", NULL, 1696960800, NULL);
-*
\ No newline at end of file
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0xA88594D404727625A9437C3f886C7643872296AE", "Moonwell", "C");INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0xA88594D404727625A9437C3f886C7643872296AE", "A", 8453, "0xA88594D404727625A9437C3f886C7643872296AE", 18, NULL);INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0xA88594D404727625A9437C3f886C7643872296AE", "WELL", "moonwell-artemis", "well", NULL, 1704752551, "eip155:8453/erc20:0xFF8adeC2221f9f4D8dfbAFa6B9a297d17603493D");
```

Comment:
you got it wrong. It should be an update statement in the old token saying `swapped_for=0xA88594D404727625A9437C3f886C7643872296AE`. From the api docs `An optional attribute representing another asset for which this asset was swapped for`